sdk/appcenter/src/androidTest/java/com/microsoft/appcenter/utils/storage/DatabaseManagerAndroidTest.java

```diff
@@ -408,6 +408,9 @@ public void upsert() {
             ContentValues contentValues = generateContentValues();
             databaseManager.upsert(contentValues);
             assertEquals(1L, databaseManager.getRowCount());
+            ContentValues actual = get(databaseManager, 1);
```

Comment:
the replace function will work when the primary key is set, explicitly use contentValues.put("oid", 1) and call the replace will work too